web.rb

```diff
@@ -173,13 +177,14 @@ def authenticate!
   status 200
 end
 
-def create_payment_intent(amount, source_id, customer_id = nil,
+def create_payment_intent(amount, source_id, payment_method_id, customer_id = nil,
                           metadata = {}, currency = 'usd', shipping = nil)
   return Stripe::PaymentIntent.create(
     :amount => amount,
     :currency => currency || 'usd',
     :customer => customer_id,
     :source => source_id,
+    :payment_method => payment_method_id,
```

Comment:
hmm, is it documented what happens here if you pass a source and a payment_method?